### PR TITLE
Added a quick blurb to let folks know where they can check pending maintenance updates

### PR DESCRIPTION
**Pull request by josh-martin-aiven**
# What changed, and why it matters

After a recent case with a customer, I noticed that it would be helpful to include some links in our documentation on maintenance updates. These links show customers where they can check their service for pending maintenance updates. 

I think it's important to have something like this in the maintenance docs, as it is not readily apparent or searchable how to get this information from a customer perspective. 

I tried to keep this update concise, and focused on directing customers to existing documentation on APIs as opposed to putting direct usage info on this doc. 

Let me know your thoughts on this addition. I'm happy to discuss any and all feedback. :) 

# Reference links
Support task Jira: 
https://aiven.atlassian.net/browse/SUP-717

Internal case study:
https://aiven.slab.com/posts/where-can-customers-check-for-pending-maintenance-updates-00047810-j9zh17em

### CODE REVIEW

File: docs/platform/concepts/maintenance-window.rst [hunk 1]
```diff
@@ -24,4 +24,12 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
...
\ No newline at end of file
+Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you.
+
+Pending maintenance updates can be viewed in three places.
```
Comment (wojcik-dorota) [inline docs/platform/concepts/maintenance-window.rst:29]: Please use the active voice (vs the passive voice) whenever possible:
You can view maintenance updates pending for your service using the following:

File: docs/platform/concepts/maintenance-window.rst [hunk 2]
```diff
@@ -24,4 +24,12 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
...
\ No newline at end of file
+Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you.
+
+Pending maintenance updates can be viewed in three places. 
+
+1. Service overview page in `the Aiven Console <https://console.aiven.io/>`_.
```
Comment (wojcik-dorota) [inline docs/platform/concepts/maintenance-window.rst:31]: Please
* Rephrase to ``Service's **Overview** page in `Aiven Console <https://console.aiven.io/>`_`` (GUI items always within ``**``)
* Use the unordered list (bullets) (the order of the items in the list is not significant).
* Remove the full stops at the ends of all three list items (the list items are not full sentences).

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]
This is very helpful content, thanks for adding this! A few editorial remarks.

File: docs/platform/concepts/maintenance-window.rst (hunk 1 above)
Comment (josh-martin-aiven) [reply docs/platform/concepts/maintenance-window.rst]: Doneso. :)

File: docs/platform/concepts/maintenance-window.rst (hunk 2 above)
Comment (josh-martin-aiven) [reply docs/platform/concepts/maintenance-window.rst]: Also done. 

Also, I finally found the style guide. I'll make sure to review this for future docs updates. If this isn't the right style guide, lemme know. :) 
https://aiven.slab.com/posts/style-guide-resources-for-editors-dag2nrwl

Comment (wojcik-dorota) [reply docs/platform/concepts/maintenance-window.rst]: Actually the most relevant is https://github.com/aiven/devportal/blob/main/CONTRIBUTING.rst :-)

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]
Thanks! a few more details :-)

File: docs/platform/concepts/maintenance-window.rst [hunk 3]
```diff
@@ -24,4 +24,12 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
...
\ No newline at end of file
+Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you.
+
+You can view pending maintenance updates in three places.
```
[the remark this answers is not in the document]
Comment (josh-martin-aiven) [reply docs/platform/concepts/maintenance-window.rst]: Got it. Just updated the wording to match latest suggestions. Let me know if there's anything else you'd like to adjust, discuss. :)

**Review by wojcik-dorota — APPROVED** [review]
Thanks for all the updates! LGTM now :-)

**Pull request merged**